### DISCUSSION TITLE
Lua language server support for arm

### DISCUSSION THREAD

**Discussion by Iamafnan** [termux/termux-packages#8487] [Ideas]
Hey is there any plan to add support of arm?

**Comment by xtkoba** [comment]
I'll give it a try.

**Comment by MrAdityaAlok** [comment]
@Iamafnan added in #8493 
You can test from ci artifacts (arm) https://github.com/termux/termux-packages/suites/4869859962/artifacts/138898513

**Comment by Iamafnan** [comment]
Hey @MrAdityaAlok i have tested the artifact and it's working good
